core/schemas/package.py

```diff
@@ -0,0 +1,276 @@
+import json
+from datetime import datetime, timezone
+from typing import Any, ClassVar, Dict, List, Optional, Tuple
+
+from pydantic import BaseModel, Field, model_validator
+from typing_extensions import Self
+
+from core.schemas import entity, indicator, observable
+from core.schemas.observable import ObservableTypes
+
+
+class YetiPackageRelationship(BaseModel):
+    target: str
+    link_type: str = "observes"
+
+
+class YetiPackage(BaseModel):
+    timestamp: str | int  # add validator
+    source: str = Field(min_length=3)
+    tags: Optional[List[str]] = []
```

Comment:
Another case for the docstring: explaining how this will be used. AFIAU it's going to tag *all* objects created, so maybe rename to "global_object_tags"?

Reply:
We could also have something like:

```python
tags: Optional[Dict[str, List[str]]]
```

where dict key would correspond to one of the package object and we could also have a special key `global` to add defined tags to all objects in the package.

```python
{
   "tags": {
      "obs1": ["tag1", "tag2"],
      "obs2": ["tag1"],
      "entity1": ["yet_another_tag"],
      "global": ["global_tag"],
}
```

Reply:
Yup that sounds good!

Reply:
Fixed in 2a2b136845b1fe077338c52f0464063841f3dfa1